src/main/java/com/homework/mateakademy/controller/MainController.java

```diff
@@ -0,0 +1,71 @@
+package com.homework.mateakademy.controller;
+
+
+import com.homework.mateakademy.domain.Message;
+import com.homework.mateakademy.domain.User;
+import com.homework.mateakademy.repositories.MessageRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.Map;
+
+@Controller
+public class MainController {
+
+    private MessageRepository messageRepo;
+
+    @Autowired
+    public DI(MessageRepository messageRepo) {
```

Comment:
method name in lowerCamelCase

Reply:
use dependency injection with lombok
https://www.baeldung.com/spring-injection-lombok